### PR TITLE
fix flutter install command : pub get -> pub add

### PR DESCRIPTION
**Pull request by imtushardave**
## What does this PR do?

Fix flutter installation command - pub get to pub add

## Related PRs and Issues

* https://github.com/appwrite/appwrite/issues/5277

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
Yes

### CODE REVIEW

File: templates/dart/README.md.twig [hunk 1]
```diff
@@ -35,7 +35,7 @@ dependencies:
 You can install packages from the command line:
 
 ```bash
-pub get {{ language.params.packageName }}
+pub add {{ language.params.packageName }}
```
Comment (stnguyen90) [inline templates/dart/README.md.twig:38]: This won't actually run successfully. It should be:

```suggestion
dart pub add {{ language.params.packageName }}
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

File: templates/flutter/README.md.twig [hunk 2]
```diff
@@ -35,7 +35,7 @@ dependencies:
 You can install packages from the command line:
 
 ```bash
-pub get {{ language.params.packageName }}
+pub add {{ language.params.packageName }}
```
[the remark this answers is not in the document]
Comment (imtushardave) [reply templates/flutter/README.md.twig]: Hi @stnguyen90 , Fixed this.

File: templates/dart/README.md.twig (hunk 1 above)
Comment (imtushardave) [reply templates/dart/README.md.twig]: Fixed

**Review by stnguyen90 — APPROVED** [review]
Hey 👋 awesome work on your PR! We've approved your work and it'll be merged soon!

**Review by lohanidamodar — APPROVED** [review]

**Pull request merged**